crates/lib/Cargo.toml

```diff
@@ -21,16 +21,19 @@ required-features = ["commandline"]
 [lib]
 name = "grass"
 path = "src/lib.rs"
-# crate-type = ["cdylib", "rlib"]
+crate-type = ["cdylib", "rlib"]
```

Comment:
I'm a bit dubious of this change, since this would affect the crate regardless of whether it's being compiled for wasm. Does `cargo run` still work with this crate?

Reply:
The WASM build fails if I do not specify this...

https://github.com/JavaScriptPlayground/grass/actions/runs/7883205359/job/21509713121#step:4:1

Reply:
Yeah I believe you'd need this line for the wasm build, but then does having this line affect the non-wasm build? For context, this line is commented out in the Cargo.toml because for the existing WASM builds this line would be manually modified before a release. We wouldn't want consumers on crates.io to have this dependency randomly be a dynamic library.

Reply:
I am not really sure if it does have an effect on non-wasm build... I mean all the tests do pass.